index.html

```diff
@@ -1,189 +1,227 @@
+                     to discover and enjoy 360 photos and videos on Gear VR.
+                    </p>
+                <ul class="trainning">
+                    <li class="other-skills">html</li>
+                    <li class="ruby">Ruby on rails</li>
+                    <li class="other-skills">css</li>
+                    <li class="other-skills">javascript</li>
+                </ul>
+                <button class="card-btn">See Project</button>
+            </div>
+        </div>
+            <div id="for" class="card">
+                <div class="pic">
+                    <img   class="pic" src="2.png" alt="">
+                </div>
+            <div>
+                <h2 class="title">Uber Navigation</h2>
+                    <ul class="field">
+                        <li class="cano">Uber</li>
+                        <li class="tag">&#8226; Lead Developer </li>
+                        <li class="tag">&#8226; 2018</li>
+                    </ul>
+                    <p class="instruction">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                <p class="desk-praa"> 
+                    A smart assistant to make driving more safe, 
+                    efficient, and fun by unlocking your most expensive computer: your car.
+                </p>
+                <ul class="trainning">
+                    <li class="other-skills">html</li>
+                    <li class="ruby">Ruby on rails</li>
+                    <li class="other-skills">css</li>
+                    <li class="other-skills">javascript</li>
+                </ul>
+                <button class="card-btn">See Project</button>
+            </div>
+        </div>
+            </section>
+            <section class="about-myself">
+                <div class="content">
+                <h2 class="myself">About Myself</h2>
+                <p class="about-para"> Hello I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+                <h2 class="about-sub-head">Let's Connect</h2>
+                    <ul id="social-media">
+                        <li><a href="#" class="social"><img src="twitter.svg" alt="twitter"></a></li>
+                        <li><a href="#" class="social"><img src="linkdn.png" alt="linkdn-icon"></a></li>
+                        <li><a href="#" class="social"><img src="m.svg" alt="media-con"></a></li>
+                        <li><a href="#" class="social"><img src="hand.svg"alt="github-con"></a></li>
+                        <li><a href="#" class="social"><img src="git.svg"></a></li>
+                    </ul>
```

Comment:
- [ ] Kindly adjust  the socail icons on the desktop view to be aligned with the paragrah and text around it.
- Look at the image comparison bellow to have a very detailed view. This gives a better user experince and will enhance your attention to details in replating designs given 👍 

## Your design:
![image](https://github.com/mumbereh/my-portfolio/assets/50990925/1d6e534d-5d14-41ac-88a3-102898a03ac8)

## Figma:
![image](https://github.com/mumbereh/my-portfolio/assets/50990925/46eed7ee-54ab-495c-b701-113abf773bab)